app/nasa_collaboration/page.js

```diff
@@ -1,30 +1,59 @@
 "use client";
 
-import React, { useState, useEffect } from 'react';
-import styles from './page.module.css';
+import React, { useState, useEffect } from "react";
+import styles from "./page.module.css";
 
 // Read "/app/nasa_collaboration/README.md" for more info about the API_KEY
 // You need a proper API_KEY for the requests to work
-const API_KEY = 'API_KEY';
+const API_KEY = "nDgmi28bavvf27vTGSZ8JoPwd4BfLJtrJP6DKhgC";
```

Comment:
For security, it is advised not to leave API keys in code that is committed to a repository.
One solution is using a `.env` file that is then added to your Git ignore list. See https://medium.com/@rivaifnasution/tutorial-how-to-use-dotenv-in-react-for-beginners-ac79d9b6ddbf